pandas/tests/io/formats/test_style.py

```diff
@@ -1,17 +1,20 @@
 import copy
-import textwrap
 import re
+import textwrap
 
-import pytest
 import numpy as np
+import pytest
+
+import pandas.util._test_decorators as td
+
 import pandas as pd
 from pandas import DataFrame
 import pandas.util.testing as tm
-import pandas.util._test_decorators as td
 
-jinja2 = pytest.importorskip('jinja2')
 from pandas.io.formats.style import Styler, _get_level_lengths  # noqa
```

Comment:
This is the reason the build is failing, the `jinja2 = pytest.importorskip('jinja2')` should come before.
Should I reorder the lines and add `# isort:skip`?

Reply:
I'm wondering why the `importerskip` is needed and why we don't add jinja as a dependency for the relevant builds, for now I think skipping is a fair idea unless anyone has ideas regarding my first point.

Reply:
Alright, I'll try with the `isort:skip`